docs/articles/documentation/reference/test-api/testcontroller/setfilestoupload.md

```diff
@@ -5,7 +5,7 @@ permalink: /documentation/reference/test-api/testcontroller/setfilestoupload.htm
 ---
 # t.setFilesToUpload Method
 
-Populates the specified file upload input with file paths.
+Populates the specified file upload element with file paths. The element must be an `<input>` with the `type="file"` attribute.
```

Comment:
> Populates the specified file upload element with file paths.

Emulate the choosing specified files to upload with browser's 'Choose File' dialog.

Reply:
> Selects files to upload. Emulates a user's selection in the browser's 'Choose File' dialog.